src/client/locales/default.json

```diff
@@ -32,10 +32,13 @@
   "bookmarks": "Bookmarks",
   "bookmarks_empty": "You don't have any story saved.",
   "notifications": "Notifications",
-  "notification_following_username": "{username} is now following you.",
-  "notification_mention_username_post": "{username} mentioned you on this post {post}.",
-  "notification_reply_username_post": "{username} replied you on your {post}.",
+  "notification_following_username": "{username} started following you.",
```

Comment:
Don't use dot `.` in the end of these sentences.

Reply:
Also I dont see this sentence been used
![image](https://user-images.githubusercontent.com/16245250/36410376-4e75d69a-1643-11e8-9f73-ca709b2ba317.png)

Reply:
you don't see those sentences being used because it goes to the en-US.json file, do you want me to change it there as well? I remember you guys telling me not to change it there. @bonustrack